perma_web/perma/tasks.py

```diff
@@ -636,6 +634,9 @@ def get_srcset_image_urls(dom_tree):
             src = src.strip().split()[0]
             if src:
                 urls.append(src)
+    # get src, too: Chrome doesn't do this automatically, although phantomjs does
```

Comment:
Should this behind a flag, ` if settings.CAPTURE_BROWSER == 'Chrome':` or ` if settings.CAPTURE_BROWSER != 'PhantomJS':` or similar? Maybe not sufficiently expensive to care, but maybe?

Reply:
Yes, commit for this is in the pipeline. Thx!